server/models/types.ts

```diff
@@ -121,13 +121,21 @@ export type AnomalyResult = {
   confidence: number;
   entity?: Entity[];
   features?: { [key: string]: FeatureResult };
+  contributions: { [key: string]: FeatureContributionData };
```

Comment:
can make this optional

Reply:
kinda did this on purpose, this is a pretty nested object. Making it optional makes the parsing process later difficult

Reply:
Older anomaly results may be read by this plugin though, where `contributions` does not exist.